source/docs/04-cmn.xml

```diff
@@ -1193,7 +1210,29 @@
                   <p>Repetition is a characteristic feature of music. Many musical forms rely on repetition (sometimes with modification) of distinct sections of the music. Repetition in this sense can be thought of as ‘structural’. At the same time, composers and engravers of music often use local symbols for repeating smaller portions of music instead of writing them in full more than once. In this case, the repetition is better defined as a species of abbreviation.</p>
                   <div xml:id="cmnRepStruct" type="div4">
                      <head>Structural Repetition</head>
-                     <p>Large-scale structural repetition, utilizing <gi scheme="MEI">section</gi> and <gi scheme="MEI">expansion</gi> elements, is discussed in section <ptr target="#sharedMdivContent"/>. This section will focus on repetition within sections.</p>
+                     <p>The most common variety of structural repetition is the use of repeat bar lines and endings. The encoding of bar lines is discussed in section <ptr target="cmnBarlines"/>, and endings in <ptr target="#sharedStrucRep"/>.</p>
+                     
+                     <p>For larger-scale repetition using da capo, segnos, codas, and etc., MEI provides the <gi scheme="MEI">repeatMark</gi> element:</p>
```

Comment:
```suggestion
                     <p>The most common variety of structural repetition is the use of repeat bar lines and endings. The encoding of bar lines is discussed in section <ptr target="cmnBarlines"/>, and that of large-scale structural repetition, utilizing <gi scheme="MEI">section</gi>, <gi scheme="MEI">ending</gi> and <gi scheme="MEI">expansion</gi> elements, is discussed in section <ptr target="#sharedStrucRep"/>.</p>
                     <p>This section will focus on repetition within sections using repetition marks like da capo, segnos, codas, and etc. For this puprose, MEI provides the <gi scheme="MEI">repeatMark</gi> element:</p>
```

Reply:
Above is meant as a proposal to keep the reference to the large-scale struReps in ch. 2

Reply:
I'm not sure I follow the intent of this change? My general understanding is that repeat bars and endings are typically smaller-scale than the marks represented by `<repeatMark>`, so I find the reference to endings being "large-scale" confusing. Also, `<section>` and `<expansion>` do not encode any marks found in music encoding directly, so I think readers may find their mention here confusing among references to encoded marks like repeat barlines and endings.